test/bdd/fixtures/scripts/rp-rest_start.sh

```diff
@@ -19,7 +19,7 @@ registerRPTenant() {
         response=$(curl -k -o - -s -w "RESPONSE_CODE=%{response_code}" \
         --header "Content-Type: application/json" \
         --request POST \
-        --data '{"label": "rp.trustbloc.local", "callback": "'$callbackURL'", "scopes": ["CreditCardStatement"]}' \
+        --data '{"label": "rp.trustbloc.local", "callback": "'$callbackURL'", "scopes": ["CreditCardStatement","DrivingLicense"]}' \
```

Comment:
Can we have a separate `RP Tenant` for bank account rather than mixing with existing one? Not a blocker but probably a next PR.

Reply:
@rolsonquadras what would be the rationale?

With the way it's set up, it's just the UI component that needs to specify the scope, which is a good thing.

Reply:
The 'Bank account' RP is a separate RP than the 'Apply CreditCard' - Two RPs and Two RP Adapter Tenant.

Also, It would be nice to demo our adapter capability to show support for multiple 'RP Adapter'.